src/Processing.js

```diff
@@ -127,15 +127,13 @@
     // JavaScript event binding and releasing
     ////////////////////////////////////////////////////////////////////////////
 
-    var eventHandlers = [];
```

Comment:
Hm, is this change necessary? The variable scoping is already 1-to-1, with one `p` instance matched to a single `eventHandlers` in this code - keeping the eventHandlers list off of the instance itself prevents potential collisions with user code bindings, so I'd strongly recommend keeping this here, rather than adding it onto `p`.

From looking at the code, just having the `detachEventHandlersByType` update on line 173 *should* still do the trick.